src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -1462,6 +1462,7 @@ def generate(
         task=None,
         language=None,
         is_multilingual=None,
+        initial_prompt_ids: Optional[List[int]] = None,
```

Comment:
Sounds good, renamed to prompt_ids. 

With regards to the type, I wonder if this is ok here because `prompt_ids` purpose is just to update `forced_decoder_ids` and `decoder_start_token_id` which are type int and List[List[int]] (if I'm not mistaken, saw that [here](https://github.com/huggingface/transformers/blob/48fbd8fa2ebfcf35c14f3bca4055631bf83e7e8b/src/transformers/generation/configuration_utils.py#L178))? If we used torch.Tensor, we would also have to add importing torch to the file with the `get_prompt_ids` function and I don't believe we'd ever require tensor functionality. Just lmk whichever you prefer